### DISCUSSION TITLE
Future Roadmap

### DISCUSSION THREAD

**Discussion by nouman91** [jpmorganchase/modular#542] [General]
Thank you for this solution, I have been working on a similar solution for the past couple of months. Almost all the solution/discussions out there are geared towards packages publishing, but none for apps and applications. But modular really solves the problem for apps. 

We are still in the early stages of development, I am figuring a few things out, but I am curious what is the future roadmap of this project and how much change it will be from the existing solution? What will be support like for new releases of CR. 

Looking forward to your response.

**Comment by threepointone** [comment]
Hiya! Sorry for the delay in responding. We're working on getting to v1 at the moment (tracking here). We're actively tracking CRA updates (hence the support for v4 at the moment). That said, we're also actively looking at what's happening with vite/snowpack/esbuild at al. Finally, we have a whole bunch of docs to write. You're welcome to use this, but we may break minor feature occasionally before v1.
